photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditor.java

```diff
@@ -811,6 +922,8 @@ public void onViewAdd(BrushDrawingView brushDrawingView) {
 
     @Override
     public void onViewRemoved(BrushDrawingView brushDrawingView) {
+        final List<View> addedViews = viewState.getAddedViews();
```

Comment:
Why we are adding this in every callback?

Reply:
Changed so the logic for accessing `addedViews` is moved into `PhotoEditorViewState`.